da/da_test.go

```diff
@@ -97,26 +92,21 @@ func TestSubmitRetrieve(t *testing.T) {
 	}
 }
 
-func startMockGRPCServ() *grpc.Server {
-	srv := proxy.NewServer(goDATest.NewDummyDA(), grpc.Creds(insecure.NewCredentials()))
-	lis, err := net.Listen("tcp", "127.0.0.1"+":"+strconv.Itoa(7980))
+func startMockDAServ() *proxy.Server {
+	srv := proxy.NewServer("localhost", "7980", goDATest.NewDummyDA())
+	err := srv.Start(context.TODO())
```

Comment:
same, background context, or we should create the context in the test and pass it into this helper. That is probably the better way to handle it.